### ISSUE TITLE
Implement API: Subscribe and Unsubscribe from push services

### ISSUE THREAD

**Issue by simonrozsival** [toggl/mobileapp#5033]
The server needs to know token of the device which we obtained from #5030 and #5031 in order to send any PNs to it.

We must wait for the backend team to create the endpoint specification before we can start working on this issue.

**Comment by simonrozsival** [comment]
Implementation was merged into the project branch.
